### PR TITLE
test(): e2e tests for npm owner

### PR DESCRIPTION
**Pull request by Izak88**

### CODE REVIEW

File: tests/e2e/setup/400-morose-init-files.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  let config = {
+    port: 10000,
+    ssl: false,
+    secret: secret,
+    upstream: 'https://registry.npmjs.org',
+    useUpstream: true,
+    saveUpstreamPackages: true
+  };
+  let auth = {
+      organizations: [
```
Comment (jkuri) [inline tests/e2e/setup/400-morose-init-files.ts]: indentation ... till line 32, whole block.

File: tests/e2e/setup/400-morose-init-files.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      users: [
+        { name: 'admin', password: pass, fullName: '', email: '', tokens: [] },
+        { name: 'developer', password: pass, fullName: '', email: '', tokens: [] }
+      ],
+      packages: []
+    };
+
+  return Promise.resolve()
+    .then(() => fs.createDir('morose'))
+    .then(() => {
```
Comment (jkuri) [inline tests/e2e/setup/400-morose-init-files.ts]:
```ts
.then(() => fs.writeJsonFile(configPath, config))
.then(() => fs.writeJsonFile(authPath, auth))
```
this is nicer, I think. 
When using arrow function `=>` you can use oneliners, they return by default, so:
```ts
.then(() => {
  return fs.writeJsonFile(configPath, config);
})
```
is exactly same as
```ts
.then(() => fs.writeJsonFile(configPath))
```

File: tests/e2e/setup/400-morose-init-files.ts (hunk 1 above)
Comment (Izak88) [reply tests/e2e/setup/400-morose-init-files.ts]: done

**Review by jkuri — CHANGES_REQUESTED** [review]
as usual, only code style glitches.. otherwise nice tests 👍

**Comment by Izak88** [comment]
changes after review done.

**Review by jkuri — APPROVED** [review]

**Pull request merged**